overrides/common/story-data.ts

```diff
@@ -0,0 +1,35 @@
+export const themeLandingPageIds = [
```

Comment:
@agurvich We can use this file to save story ids per each page (I just subbed the data whatever value for now)